splunk_add_on_ucc_modinput_test/common/splunk_instance.py

```diff
@@ -70,14 +70,19 @@ def _validate_index_name(index_name: str) -> None:
 
     @staticmethod
     def _victoria_create_index(
-        index_name: str, *, acs_stack: str, acs_server: str, splunk_token: str
+        index_name: str,
+        datatype: str,
```

Comment:
Could you move **datatype** after *. This will force user to use key-value style for datatype when calling this method. So if beside index_name user specifies any configuration it will be. seen what exactly is configured